packages/blade-old/src/atoms/TextInput/AccessoryIcon.web.js

```diff
@@ -64,7 +63,7 @@ const AccessoryIcon = ({ name, disabled, hasError, variant, isFocused, position
 };
 
 AccessoryIcon.propTypes = {
-  name: PropTypes.string,
+  icon: PropTypes.elementType,
```

Comment:
We can do that, but then again it will make the whole refactor pointless since we have to import all the icons to check (on the runtime) to see if they are the icons we provide. bundlesize issue will remain in that scenario.